Makefile

```diff
@@ -134,9 +134,6 @@ export OUT_DIR=$(GO_TOP)/out
 export ISTIO_OUT:=$(GO_TOP)/out/$(GOOS)_$(GOARCH)/$(BUILDTYPE_DIR)
 export HELM=$(ISTIO_OUT)/helm
 
-# istioctl kube-inject uses builtin config only if this env var is set.
```

Comment:
Look at [`kubeinject.go`](https://github.com/istio/istio/blob/master/istioctl/cmd/istioctl/kubeinject.go#L173) it mentions the builtin sidecar template in several places.

Can you update references and documents in that file as well?

Reply:
Done.